cmd/imagec/imagec.go

```diff
@@ -395,7 +402,10 @@ func CreateImageConfig(images []*ImageWithMeta) error {
 			Comment:   image.Comment,
 		}
 		history = append(history, h)
-		rootFS.DiffIDs = append(rootFS.DiffIDs, dockerLayer.DiffID(layer.diffID))
+		diffID := layer.diffID
+		rootFS.DiffIDs = append(rootFS.DiffIDs, dockerLayer.DiffID(diffID))
+		diffIDs[layer.diffID] = layer.ID
```

Comment:
Ahh, good catch. That was a remnant of a previous change that didn't get fully refactored. Fixed.